HABApp/openhab/http_connection.py

```diff
@@ -320,6 +331,35 @@ async def async_get_item(self, item_name: str) -> dict:
         else:
             return await ret.json(encoding='utf-8')
 
+    async def async_thing_remove_link(self, thing_uid: str, channel: str, item_name: str) -> bool:
+        if self.config.general.listen_only:
+            return False
+
+        fut = self.__session.delete(self.__get_link_url(thing_uid, channel, item_name), json={})
+        
+        ret = await self._check_http_response(fut)
+        return ret.status == 200
+
+    async def async_thing_link_exists(self, thing_uid: str, channel: str, item_name: str) -> bool:        
+        fut = self.__session.get(self.__get_link_url(thing_uid, channel, item_name), json={})
+
+        ret = await self._check_http_response(fut, accept_404=True)
+        return ret.status == 200
+
+    async def async_thing_add_link(self, thing_uid: str, channel: str, item_name: str) -> bool:
+        if self.config.general.listen_only:
+            return False
+
+        # check for item existence first, otherwhise OpenHAB creates a new item when we add a link with an unknown itemname
+        exists = await self.async_item_exists(item_name)        
+        if not exists:
+            raise ItemNotFoundException(f'Item "{item_name}" does not exist')
+        
+        fut = self.__session.put(self.__get_link_url(thing_uid, channel, item_name), json={})
```

Comment:
Is the parameter json really required?
```suggestion
        fut = self.__session.put(self.__get_link_url(thing_uid, channel, item_name))
```